docs/Getting Started/Auto-Suggest.md

```diff
@@ -59,7 +59,7 @@ Here is a more detailed walk through of the creation of a new task, which can be
 
 7. Now our task is complete:
 
-    ![auto-suggest-menu-after-due-emoji](../images/auto-suggest-menu-after-due-emoji.png)
+    ![auto-suggest-menu-after-due-emoji](../images/auto-suggest-menu-after-due-date.png)
```

Comment:
I noticed the image was repeated in the documentation, so I took the opportunity to change it to what seems like the correct one instead.

Reply:
OK, thanks. Oh, that's much better.

The alt-text for screen-readers was rubbish before, but please could you update it as well, to at least make It consistently rubbish? 🤣

Reply:
I did not notice the alt text! 😂 Fixing it!